bindata/network/network-metrics/002-prometheus.yaml

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    name: monitor-network
+  annotations:
+    networkoperator.openshift.io/ignore-errors: ""
+  name: monitor-network
+  namespace: openshift-multus
+spec:
+  endpoints:
+    - interval: 10s
+      port: metrics
+      metricRelabelings:
+        - sourceLabels: [pod]
```

Comment:
this looks like you just want to `honorLabels: true`

Reply:
Ah, exactly. Thanks!

Reply:
Done, this needs the change on the daemon itself to get merged.